services/gem/gem-version.service.js

```diff
@@ -35,8 +57,24 @@ module.exports = class GemVersion extends BaseJsonService {
     })
   }
 
-  async handle({ gem }) {
-    const { version } = await this.fetch({ gem })
-    return this.constructor.render({ version })
+  async fetchLatest({ gem }) {
+    return this._requestJson({
+      schema: versionSchema,
+      url: `https://rubygems.org/api/v1/versions/${gem}.json`,
+    })
+  }
+
+  async handle({ gem }, queryParams) {
+    if (queryParams && typeof queryParams.include_prereleases !== 'undefined') {
+      const data = await this.fetchLatest({ gem })
+      if (!Array.isArray(data) || data.length === 0) {
```

Comment:
I'm a little confused here: shouldn't the `versionSchema` guarantee that we can't hit this case?

Reply:
Yes, that actually makes sense, having this check is redundant. I'll get rid of it, thanks!